specifyweb/specify/management/commands/run_key_migration_functions.py

```diff
@@ -0,0 +1,116 @@
+import logging
+from django.core.management.base import BaseCommand
+from django.apps import apps
+from django.db import transaction
+from specifyweb.backend.businessrules.migration_utils import catnum_rule_editable
+from specifyweb.backend.businessrules.uniqueness_rules import (
+    apply_default_uniqueness_rules,
+    create_uniqueness_rule
+)
+from specifyweb.permissions.migration_utils.edit_permissions import add_permission, add_stats_edit_permission
+from specifyweb.specify.migration_utils.default_cots import (
+    create_cogtype_type_picklist,
+    create_cotype_picklist,
+    create_default_collection_types,
+    create_default_discipline_for_tree_defs,
+    fix_taxon_treedef_discipline_links,
+    set_discipline_for_taxon_treedefs,
+    fix_tectonic_unit_treedef_discipline_links
+)
+from specifyweb.backend.permissions.initialize import initialize
+from specifyweb.specify.migration_utils import update_schema_config as usc
+from specifyweb.specify.migration_utils.misc_migrations import make_selectseries_false
+from specifyweb.specify.migration_utils.tectonic_ranks import create_default_tectonic_ranks, create_root_tectonic_node
+from specifyweb.backend.patches.migration_utils import apply_migrations as apply_patches
+
+logger = logging.getLogger(__name__)
+
+def fix_cots():
+    create_default_collection_types(apps)
+    # create_default_discipline_for_tree_defs(apps)
+    create_cogtype_type_picklist(apps)
+    set_discipline_for_taxon_treedefs(apps)
+    fix_taxon_treedef_discipline_links(apps)
+    create_cotype_picklist(apps)
+
+def fix_schema_config():
+    usc.create_geo_table_schema_config_with_defaults(apps) # specify 0002
+    usc.create_cotype_splocalecontaineritem(apps) # specify 0003
+    usc.create_strat_table_schema_config_with_defaults(apps) # specify 0004 - getting skip warnings
+    usc.create_agetype_picklist(apps) # specify 0004
+    usc.update_cog_type_fields(apps) # specify 0007
+    usc.create_cogtype_picklist(apps) # specify 0007
```

Comment:
Is this redundant (or is the earlier work redundant) where it creates the default COG type pick list? Seems this should handle it AFAIK